test/basic-querying.test.js

```diff
@@ -525,31 +525,31 @@ describe('basic-querying', function() {
 
     it('should find first record (default sort by id)', function(done) {
       User.all({ order: 'id' }, function(err, users) {
-        User.findOne(function(e, u) {
-          should.not.exist(e);
-          should.exist(u);
-          u.id.toString().should.equal(users[0].id.toString());
+        User.findOne(function(err, user) {
+          should.not.exist(err);
+          should.exist(user);
+          user.id.toString().should.equal(users[0].id.toString());
```

Comment:
IIUC, this change is cleaning up the code by renaming `e, u` to `err, user`. While I see a lot of value in clean code, I see much more value in git commit history that makes code archaeology easy. Changes like these makes examination of history more difficult, as they add noise.

If you think this is still a change worth making, then please put it in a standalone commit, possibly in a new pull request.